src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -527,6 +554,8 @@ impl HydraulicLoopController for A320HydraulicLoopController {
 }
 
 struct A320EngineDrivenPumpController {
+    is_powered: bool,
+    powering_buses: Vec<ElectricalBusType>,
```

Comment:
The controller is _powered by_ an electrical bus.

```suggestion
    powered_by: Vec<ElectricalBusType>,
```